modules/_labs/core/react/spec/type.snapshot.tsx

```diff
@@ -49,23 +50,23 @@ describe('Core Labs Type Snapshot', () => {
 
         <h3 style={type.h3}>Variants</h3>
 
-        <span className={css(type.body, type.variant.button)}>Button Text</span>
+        <span css={css(type.body, type.variant.button)}>Button Text</span>
```

Comment:
Isn't this the same as:
```tsx
<span css={[type.body, type.variant.button]}>Button Text</span>
```
Otherwise, why use the jsx pragma + the `css` prop?

Reply:
You are correct, I will update these 👍